### PR TITLE
feat: Chapi save credentials UI change

### LINKED ISSUE #1061: Update store presentation/Save credential store.vue UI in wallet
https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/f6a8e451aadd78b86581ebae130dd68c8411f30c/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/D0DFC114-CF04-4E5E-B80C-3C0F91F501E6?collectionId=523ddc58-c7a7-427e-9f18-ed19b90b1b9c&collectionLayerId=c6674955-3128-4c42-aebb-1d5ebfccbdc5

### PR DESCRIPTION
**Pull request by talwinder50**
Screen Shot 2021-09-13 at 9 23 19 PM
Screen Shot 2021-09-13 at 9 32 48 PM

https://user-images.githubusercontent.com/7862595/133180120-2c11ac82-ffa6-4043-84fa-f051fb165225.mov

closes #1061

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>

### CODE REVIEW

File: cmd/wallet-web/src/pages/Store.vue [hunk 1]
```diff
@@ -97,13 +138,21 @@ export default {
...
       errors: [],
+      selectedDefault: 'Default Vault',
     };
   },
   computed: {
     isDisabled() {
       return this.storeButton;
     },
+    i18n() {
+      // ui test failure due to this, once test is fixed can be uncommented.
```
Comment (talwinder50) [inline cmd/wallet-web/src/pages/Store.vue:153]: #1075

File: cmd/wallet-web/src/pages/Store.vue [hunk 2]
```diff
@@ -97,13 +138,18 @@ export default {
...
+      showDetails: false,
       errors: [],
+      selectedDefault: 'Default Vault',
     };
   },
   computed: {
     isDisabled() {
       return this.storeButton;
     },
+    // Todo issue 1075 add ii8n support : failing UI Test
```
Comment (talwinder50) [inline cmd/wallet-web/src/pages/Store.vue:152]: ii8n support was added but UITest test was failing, have to look why #1075

File: cmd/wallet-web/src/pages/Store.vue [hunk 3]
```diff
@@ -157,52 +201,32 @@ export default {
...
+      return getCredentialType(vc.type);
+    },
+    getCredentialDisplayData: function (vc, manifestCredential) {
+      return getCredentialDisplayData(vc, manifestCredential);
+    },
+    getManifest: function (credential) {
+      const currentCredentialType = this.getCredentialType(credential);
+      return credentialDisplayData[currentCredentialType] || credentialDisplayData.fallback;
+    },
+    toggleDetails(i) {
```
Comment (talwinder50) [inline cmd/wallet-web/src/pages/Store.vue:214]: This is to only toggle the selected credential not all

File: cmd/wallet-web/src/pages/Store.vue [hunk 4]
```diff
@@ -131,14 +177,12 @@ export default {
...
-          description: vc.description,
-          icon: getVCIcon(vc.type),
-        };
+      data.verifiableCredential.map((vc) => {
+        const manifest = this.getManifest(vc);
+        const record = this.getCredentialDisplayData(vc, manifest);
+        this.records.push(record);
       });
-      console.log('this.records', JSON.stringify(this.records, null, 2));
+      console.log('records', JSON.stringify(this.records, null, 2));
```
Comment (sudeshrshetty) [inline cmd/wallet-web/src/pages/Store.vue:185]: remove this, or for now make it `debug`

**Review by sudeshrshetty — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**